### PR TITLE
Added Rameshwar Bhaskaran (zorroblue)

### PR DESCRIPTION
**Pull request by zorroblue**
Added Rameshwar Bhaskaran to README.md

### CODE REVIEW

**Comment by hargup** [comment]
@zorroblue your pull request looks good :+1: but you need to improve your commit message. You can be more specifiying the name which was added in `README.md`.

**Comment by zorroblue** [comment]
OK :)

**Comment by krishna95** [comment]
@zorroblue @hargup It looks good to me

**Comment by hargup** [comment]
@zorroblue you have to change the commit message and the title of the pull request. Use `git commit --amend`.

File: README.md [hunk 1]
```diff
@@ -22,3 +22,4 @@ and the lines starting with `*` are automatically rendered as bullet points.
 * Harsh Gupta [hargup](http://github.com/hargup)
 * Krishna Bagadia [krishna95](http://github.com/krishna95)
 * Nevin Valsaraj [routeaccess](http://github.com/routeaccess)
+* Rameshwar Bhaskaran [ zorroblue ](http://github.com/zorroblue)
```
Comment (hargup) [inline README.md]: And you should remove the spaces surrounding `zorroblue`.

**Comment by zorroblue** [comment]
I tried amending the original commit message...The changes were reflected in the local repo..How should I overwrite my commit message in the remote repo??
I tried force push ..but it reads "Everything upto date"..

**Comment by krishna95** [comment]
@zorroblue Please resolve merge conflicts

**Comment by hargup** [comment]
@zorroblue I'm closing the pull request for the time being, please feel free to reopen it when you start working on it again.

**Pull request closed without merging**